src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor_Constants.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal partial class SymbolDisplayVisitor
     {
-        private void AddConstantValue(ITypeSymbol type, object constantValue, bool preferNumericValueOrExpandedFlagsForEnum = false)
+        private void AddConstantValue(ITypeSymbol type, object constantValue, bool preferNumericValueOrExpandedFlagsForEnum, bool allowDefaultLiteral)
```

Comment:
>bool allowDefaultLiteral [](start = 125, length = 24)

It looks like all consumers calculate this value the same way. Consider inlining this calculation into this function and getting rid of the parameter.

Reply:
This was a good cleanup, thanks!